meshtastic/interfaces/ble/lifecycle_ownership_runtime.py

```
@@ -0,0 +1,685 @@
+                    getattr(iface, "_connected_publish_inflight_client", None)
+                    is connected_client
+                ):
+                    iface._connected_publish_inflight_client = None
+        post_check_snapshot = snapshot_provider(
+            connected_client,
+            connected_device_key,
+            connection_alias_key,
+        )
+        _raise_invalidated(post_check_snapshot)
+
+    def _commit_and_publish_connected(
```

Ruff (0.15.6)

[warning] Too many branches (14 > 12)

(PLR0912)